indico/htdocs/js/indico/modules/registration/form/tpls/fields/dialogs/base.tpl.html

```diff
@@ -38,7 +38,7 @@
                         ng-class="{hasError: validationStarted && optionsForm.title.$invalid}"
                         nd-init-focus
                         class="caption-input"/>
-                    <span ng-if="settings.isRequired">
+                    <span ng-if="field.inputType != 'accommodation'">
```

Comment:
I don't think we want this. `accommodation` is not the only field we want to disallow making mandatory. With this change, for instance, `label` fields will be allowed to make mandatory, which doesn't make any sense.

Instead, what we need to do is adding `settings.isRequired = false;` in the `link` method of `ndAccommodationField`.
https://github.com/indico/indico/blob/v1.9.9/indico/htdocs/js/indico/modules/registration/form/field.js#L953

In the same way we do here:
https://github.com/indico/indico/blob/v1.9.9/indico/htdocs/js/indico/modules/registration/form/field.js#L444